flyctl/flyctl.go

```diff
@@ -68,7 +73,14 @@ func initViper() {
 	viper.BindEnv(ConfigGQLErrorLogging, "GQLErrorLogging")
 
 	viper.SetEnvPrefix("FLY")
-	viper.AutomaticEnv()
+
+	for _, key := range viper.AllKeys() {
+		fmt.Printf("loading %s from env\n", key)
```

Comment:
I'm guessing this line is for debugging and can be removed?

Reply:
Doh